<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_serv_usr_set</class>
 <widget class="QDialog" name="Dlg_serv_usr_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户设置</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>体温校准</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_nibp_correct">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>记  录  仪</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv1_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>168</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ecg_ampli">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv3_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv1_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv2_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ip2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>168</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无创血压校准</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>540</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP 地 址</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv3_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_gate2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>168</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv2_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv2_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>168</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_gate1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ip4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_record">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_net_num">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG幅度</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_spb_temp_correct">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv3_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>168</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_gate3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_gate4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_spo_correct">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv3_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv1_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器3</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ip3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SPO2校准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv2_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ip1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>网  关</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>网络 号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_serv1_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
